sql/init.sql

```diff
@@ -0,0 +1,59 @@
+CREATE DATABASE performee;
+
+CREATE TABLE IF NOT EXISTS public.plans
+(
+    plan_id    SERIAL,
+    username   VARCHAR(150) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE,
+    end_date   TIMESTAMP WITH TIME ZONE,
+    comments   TEXT,
+    PRIMARY KEY (plan_id)
+);
+
+CREATE TABLE IF NOT EXISTS public.objectives
+(
+    objective_id SERIAL,
+    plan_id      INT          NOT NULL,
+    title        VARCHAR(150) NOT NULL,
+    category     VARCHAR(100) NOT NULL,
+    description  TEXT,
+    comments     TEXT,
+    PRIMARY KEY (objective_id),
+    CONSTRAINT fk_plan FOREIGN KEY (plan_id) REFERENCES plans (plan_id)
+);
+
+CREATE TABLE IF NOT EXISTS public.goals
+(
+    goal_id             SERIAL,
+    objective_id        INT          NOT NULL,
+    title               VARCHAR(150) NOT NULL,
+    description         TEXT,
+    resources           VARCHAR(500) NOT NULL,
+    acceptance_criteria VARCHAR(500) NOT NULL,
+    comments            TEXT,
+    PRIMARY KEY (goal_id),
+    CONSTRAINT fk_objective FOREIGN KEY (objective_id) REFERENCES objectives (objective_id)
+);
+
+insert into public.plans (plan_id, username, start_date, end_date, comments)
```

Comment:
This part should not be either liquibase script or init script as these objects should be created from ui. For testing purposes we can create separate folder with test data scripts.